app/components-react/shared/Tooltip.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { CSSProperties, PropsWithChildren, HTMLAttributes } from 'react';
+import { Tooltip } from 'antd';
+import styles from './Tooltip.m.less';
+import cx from 'classnames';
+
+type TTipPosition =
+  | 'topLeft'
+  | 'top'
+  | 'topRight'
+  | 'leftTop'
+  | 'left'
+  | 'leftBottom'
+  | 'bottomLeft'
+  | 'bottom'
+  | 'bottomRight'
+  | 'rightTop'
+  | 'right'
+  | 'rightBottom';
+
+interface ITooltipTipProps {
+  title: string;
+  className?: HTMLAttributes<HTMLElement> | string;
+  style?: CSSProperties;
+  lightShadow?: boolean;
+  placement?: TTipPosition;
+  content?: HTMLElement | boolean;
+}
+
+export default function ToolTip(props: PropsWithChildren<ITooltipTipProps>) {
+  const { title, className, style, lightShadow, placement = 'bottom', content } = props;
+
+  return (
+    <div className={cx(styles.tooltipWrapper, className)}>
+      <div className={cx(styles.tooltipArrow)} />
+      {lightShadow ? (
```

Comment:
im not sure i understand why there is a ternary here if the only difference is the className being applied

Reply:
Done